src/state/apis/snapshot/snapshot.ts

```diff
@@ -0,0 +1,95 @@
+import { createApi } from '@reduxjs/toolkit/dist/query/react'
+import { type AccountId, fromAccountId } from '@shapeshiftoss/caip'
+import { isEvmChainId } from '@shapeshiftoss/chain-adapters'
+import snapshot from '@snapshot-labs/snapshot.js'
+import axios from 'axios'
+import { bn, bnOrZero } from 'lib/bignumber/bignumber'
+import { findClosestFoxDiscountDelayBlockNumber } from 'lib/fees/utils'
+
+import { BASE_RTK_CREATE_API_CONFIG } from '../const'
+import type { Strategy } from './validators'
+import { SnapshotSchema, VotingPowerSchema } from './validators'
+
+type SnapshotVotingPowerArgs = AccountId[]
+type FoxVotingPowerCryptoBalance = string
+
+const SNAPSHOT_SPACE = 'shapeshiftdao.eth'
+
+export const snapshotApi = createApi({
+  ...BASE_RTK_CREATE_API_CONFIG,
+  reducerPath: 'snapshotApi',
+  endpoints: build => ({
+    getStrategies: build.query<Strategy[], void>({
+      keepUnusedDataFor: Number.MAX_SAFE_INTEGER, // never refetch these
```

Comment:
love the pragmatism here

Reply:
there is a bikeshed on this https://github.com/reduxjs/redux-toolkit/discussions/2347

Reply:
i once published an npm package with that as a version to see if it'd break npm. sadly it didnt.